modules/virt-enabling-disabling-vm-delete-protection-cli.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * virt/managing-vms/virt-enabling-disabling-vm-delete-protection.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-enabling-disabling-vm-delete-protection-cli_{context}"]
+
+= Enabling or disabling virtual machine delete protection by using the command line
+
+To prevent the inadvertent deletion of a virtual machine (VM), you can enable virtual machine delete protection by using the command line. Should you change your mind, you can easily disable delete protection for a VM.
```

Comment:
"Should you change your mind" sounds a little informal. I'd just say something more concise that it can be disabled in the future if needed.

I think we usually try to avoid "easily" too